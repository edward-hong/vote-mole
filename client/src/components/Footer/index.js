import React from 'react'
import { Link } from 'react-router-dom'
import styled from 'styled-components'

const Footer = styled.footer`
	text-align: center;
	margin: 3rem auto 1.5rem;
`

const FooterComponent = () => (
	<Footer>
		Made by Edward Hong | <Link to="/privacy-policy">Privacy Policy</Link>
	</Footer>
)

export default FooterComponent
