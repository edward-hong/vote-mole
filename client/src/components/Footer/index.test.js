import React from 'react'
import renderWithRouter from '../../utils/renderWithRouter'

import Footer from '.'

describe('Footer', () => {
	it('renders correctly', () => {
		const { container } = renderWithRouter(<Footer />)

		expect(container.firstChild).toMatchSnapshot()
	})
})
