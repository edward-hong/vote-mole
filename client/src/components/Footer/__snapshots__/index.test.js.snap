// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
.c0 {
  text-align: center;
  margin: 3rem auto 1.5rem;
}

<footer
  class="c0"
>
  Made by Edward Hong | 
  <a
    href="/privacy_policy"
  >
    Privacy Policy
  </a>
</footer>
`;
